app/controllers/api/v1/mentors_controller.rb

```diff
@@ -1,15 +1,31 @@
 class Api::V1::MentorsController < ApplicationController
 
+  def create
+    mentor = Mentor.new(mentor_params)
+    if mentor.save
+      render json: MentorSerializer.new(mentor)
```

Comment:
@tfielder looking at this again, it does need an else for anything unforeseen, like,
 `else
          render json: {}, status: 401`